IntegrationTests/binary.bats

```diff
@@ -15,5 +15,5 @@ teardown() {
     run utica update --platform iOS --valid-simulator-archs "i386 x86_64"
 
     [ "$status" -eq 0 ]
-    [ -d Carthage/Build/iOS/Firebase.framework ]
```

Comment:
how about adding a second test here? one for fat frameworks and one for XCFrameworks

Reply:
Done